### PR TITLE
Accessibility bug fixes

### PR DESCRIPTION
**Pull request by Naseer-007**
- Added missing aria label to government of Canada logo in the header component
- Added missing aria-label to button-icon
- Fixed disabled state issue on input component where eye icon would not get disabled when input disabled
- Fixed styles for disabled eye icon
- Fixed eye icon position bug
- Fixed screen reader not reading eye icon changed state bug

### CODE REVIEW

File: component-library/component-lib/src/lib/input/input.component.ts [hunk 1]
```diff
@@ -52,6 +53,7 @@ export class InputComponent implements ControlValueAccessor, OnInit {
   @Input() formGroup = this.formGroupEmpty;
 
   disabled = false;
+  currentDisabledState: boolean = false;
```
Comment (xingruoxicic) [inline component-library/component-lib/src/lib/input/input.component.ts:56]: Is this used anywhere? If not plz remove it.

Comment (Naseer-007) [reply component-library/component-lib/src/lib/input/input.component.ts:56]: oh right, that's not being used. will remove that. Thank you

File: core-library/component-styling/input-component/_input-component.scss [hunk 2]
```diff
@@ -164,7 +164,13 @@
       background: transparent;
       z-index: 1;
       padding-right: 10px;
+      color: #161617;
     }
+
+    button:disabled {
+      color: #B2B2B4;
```
Comment (xingruoxicic) [inline core-library/component-styling/input-component/_input-component.scss:171]: So we currently don't have a token for this?

**Review by xingruoxicic — APPROVED** [review]

File: component-library/component-lib/src/lib/header/header.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <header class="heading">
     <div class="grid-container" [id]="id">
-        <img src="{{imageURL}}">
+        <img src="{{imageURL}}" attr.aria-label="{{ 'Header.header-img-alt-text' | translate }}">
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/header/header.component.html:3]: Can't do this - don't want to use the translate pipe with static text in the library for a few reasons. See other text examples in the header

File: component-library/component-lib/src/lib/input/input.component.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import {
   forwardRef,
   Input,
   OnInit,
+  ViewChild,
```
Comment (kristopher-houston) [inline component-library/component-lib/src/lib/input/input.component.ts:6]: Is this used?

File: core-library/component-styling/input-component/_input-component.scss (hunk 2 above)
Comment (Naseer-007) [reply core-library/component-styling/input-component/_input-component.scss:171]: Fixed.

File: component-library/component-lib/src/lib/input/input.component.ts (hunk 4 above)
Comment (Naseer-007) [reply component-library/component-lib/src/lib/input/input.component.ts]: Removed.

File: component-library/component-lib/src/lib/header/header.component.html (hunk 3 above)
Comment (Naseer-007) [reply component-library/component-lib/src/lib/header/header.component.html:3]: will take a look.

Comment (Naseer-007) [reply component-library/component-lib/src/lib/header/header.component.html]: Fixed.

**Review by kristopher-houston — APPROVED** [review]

**Pull request merged**